public/components/report_definitions/create/create_report_definition.tsx

```diff
@@ -301,12 +304,12 @@ export function CreateReport(props: { [x: string]: any; setBreadcrumbs?: any; ht
       <EuiPageBody>
         <EuiTitle>
           <h1>
-            {i18n.translate('opensearch.reports.createReportDefinition.title', {
+            {!getNavGroupEnabled && i18n.translate('opensearch.reports.createReportDefinition.title', {
               defaultMessage: 'Create report definition',
             })}
           </h1>
         </EuiTitle>
-        <EuiSpacer />
+        <EuiSpacer size='s'/>
```

Comment:
QQ: Do we want to remove the extra spacing if the h1 is not present/empty? Let me know if this makes sense

Reply:
Good catch, fixed with the following 
{!getNavGroupEnabled && <EuiSpacer size='s' />}
Very minor look change for navigation enabled but makes sense!